api/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
```

Comment:
No need to create an app for the APIs, You should keep the apis in the respective apps, like product apis should go to the product app etc, You can create a new file / python package for the apis.

It will be like Products > API > serliazers.py, views.py, Same goes for the other apps.

Reply:
ohh okay